dipper/sources/ZFIN.py

```diff
@@ -17,7 +17,7 @@
 
 LOG = logging.getLogger(__name__)
 ZFDL = 'http://zfin.org/downloads'
-
+OBO = 'http://purl.obolibrary.org/obo/zp/src/curation'
```

Comment:
OBO is defined in curie map, it would be good to choose a non conflicting variable name so when we come across it in any dipper code we have a good sense of what it is

Reply:
Thanks Tom! Done.